docs/artifactory/groupaddusers/help.go

```diff
@@ -0,0 +1,15 @@
+package groupaddusers
+
+const Description = "Add users to a group in the JFrog Platform."
```

Comment:
Add users to a group in the JFrog Platform.
-->
Add a list of users to a group.